shell/persistent-screen.md

```diff
@@ -0,0 +1,33 @@
+## Persistent screen or tmux session on lxplus
+
+### Setting up password-less kerberos token
+
+In order for the kerberos token to be refreshed automatically, it must be possible to do so without a password.
+Therefore, we create a keytab (similar to a public ssh key) on lxplus:
+```bash
+# /usr/kerberos/sbin/ktutil
+ktutil:  add_entry -password -p USERNAME@CERN.CH -k 1 -e arcfour-hmac-md5
+Password for USERNAME@CERN.CH: 
+ktutil:  add_entry -password -p USERNAME@CERN.CH -k 1 -e aes256-cts
+Password for USERNAME@CERN.CH: 
+ktutil:  wkt USERNAME.keytab
```

Comment:
Ok, maybe we add a sentence like "type this into your terminal" and replace the # with a $ ?